### PR TITLE
added Daedalic, Koch media, and Crows Crows Crows; added Blue Byte to Ubisoft

### PR DESCRIPTION
**Pull request by mal-tee**

### CODE REVIEW

File: companies/crows-crows-crows.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "Crows Crows Crows"
+    ],
+    "address": "54 Middle Street\nSowerby Bridge\nWest Yorkshire\nHX6 3NX\nUnited Kingdom",
+    "email": "hello@crowscrowscrows.com",
+    "web": "https://crowscrowscrows.com",
+    "sources": [
+        "https://crowscrowscrows.com/legal.html",
+        "https://crowscrowscrows.com/privacy/"
+    ],
+    "suggested-transport-medium": "email",
```
Comment (baltpeter) [inline companies/crows-crows-crows.json:20]: If the email address is not privacy-specific, we don't set the stm to `email`, even if the email is explicitly mentioned in the privacy policy.

File: companies/koch-media-gmbh.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "address": "Lochhamer Str. 9\n82152 Planegg\nGermany",
+    "phone": "+49 89 24245405",
+    "fax": "+49 89 242453405",
+    "web": "https://www.kochmedia.com",
+    "sources": [
+        "https://www.deepsilver.com/en/privacy-policy/",
+        "https://www.kochmedia.com/en/privacy-policy/"
+    ],
+    "comments": [
+        "Check https://www.deepsilver.com/en/privacy-policy/ to obtain the email address of the DPO."
```
Comment (baltpeter) [inline companies/koch-media-gmbh.json:23]: :D 

We should at least specify the general email from https://www.kochmedia.com/en/imprint/ in the record. Usually, we'd leave it at that but if you want to keep the comment, that's okay.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks!

File: companies/koch-media-gmbh.json (hunk 2 above)
Comment (mal-tee) [reply companies/koch-media-gmbh.json:24]: Okay. 

Lets keep the comment. :D

**Review by baltpeter — APPROVED** [review]

**Pull request merged**